ega-charts/localega/templates/inbox-deploy.yaml

```diff
@@ -67,6 +67,8 @@ spec:
             secretKeyRef:
               name: {{ template "localega.name" . }}-cega-creds
               key: credentials
+        - name: INBOX_LOCATION
+          value: "/ega/inbox/%s/"
```

Comment:
shouldn't this value be: `{{ .Values.config.inbox_location | quote }}` ?

Reply:
Almost, it should most likely be `{{ .Values.inbox.location | quote }}/%s`